test/tests/benchmark/data_display.py

```diff
@@ -0,0 +1,43 @@
+from modules.logger import Log
+import jinja2
+import re
+import os
+
+LOG = Log(__name__)
+
+class DataDisplay():
+    def __init__(self, name):
+        self.__testname = name
+        return
+
+    def generate_graph(self):
+        # Create a template for rrdtool command
+        cmd = """rrdtool graph {{ log_path }}/memory.png \
+'DEF:used={{ log_path }}/{{ name }}/memory/memory-used.rrd:value:AVERAGE' \
+'LINE1:used#0000FF:"used"'
+        """
+        # Get timestamp in log path
+        fh = open(os.path.dirname(__file__) + '/ansible/group_vars/benchmark')
+        item = re.compile(r'test_machine_log_path:.*')
+        path = item.match(fh.read()).group()
+        timestamp = path[(path.find('benchmark') + len('benchmark') + 1): (len(path) - 1)]
+        log_path = os.path.expanduser('~') + '/benchmark/' + timestamp
+        fh.close()
+
+        # Render the rrdtool command
+        rrdcmd_template = jinja2.Template(cmd)
+        rrdcmd = rrdcmd_template.render({
+            'log_path': log_path,
+            'name': self.__testname
+        })
+
+        os.system(rrdcmd)
+
+    def generate_comparative_graph(self):
+        return
```

Comment:
Looks like just a place holder function here. You may use `pass` instead of `return`